### PR TITLE
[Library] Add Spruce

### PR DESCRIPTION
**Pull request by ryangjchandler**
This PR adds [Spruce](https://github.com/ryangjchandler/spruce), the defacto global state management layer for Alpine.js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The TALL stack is a full stack development solution featuring some of the librar
 - [Laravel TALL Preset](https://github.com/laravel-frontend-presets/tall) - A front-end preset for Laravel to scaffold an application using the TALL stack.
 - [Laravel TALL Preset](https://github.com/use-preset/laravel-tall/) - A preset for installing the TALL stack with just one command.
 - [Laravel Views](https://github.com/Gustavinho/laravel-views) - Laravel package to create beautiful common views like data tables using the TALL stack. 
+- [Spruce](https://github.com/ryangjchandler/spruce) - A global state management layer for Alpine.js
```
Comment (innocenzi) [inline README.md:38]:
```suggestion
+ [Spruce](https://github.com/ryangjchandler/spruce) - A global state management layer for Alpine.js.
```

**Comment by ryangjchandler** [comment]
Thanks @innocenzi - missed that!

**Comment by driesvints** [comment]
Hey @ryangjchandler, this one seems very specific to Alpine.js so it might be better be off in the awesome list of Alpine.js itself. I'm looking specifically for resources that combine parts of the TALL stack. Sorry.

Of course feel free to reason why you'd like to see this merged.

**Comment by ryangjchandler** [comment]
> Hey @ryangjchandler, this one seems very specific to Alpine.js so it might be better be off in the awesome list of Alpine.js itself. I'm looking specifically for resources that combine parts of the TALL stack. Sorry.
> 
> 
> 
> Of course feel free to reason why you'd like to see this merged.

I maintain the Alpine awesome list, it's already on there. Alpine is also part of the TALL stack and global state is a common need in front-end applications so I figured it made sense to add to this list. Don't worry though.

**Comment by driesvints** [comment]
We link through to all the individual awesome lists for each piece of the stack so I'm sure people will find your library. I'm trying to keep this one a bit more focussed so it doesn't duplicates everything from each individual list.

**Comment by ryangjchandler** [comment]
> We link through to all the individual awesome lists for each piece of the stack so I'm sure people will find your library. I'm trying to keep this one a bit more focussed so it doesn't duplicates everything from each individual list.

No worries, completely understand. I've got some items to PR in anyway.

**Pull request closed without merging**